gnucash/import-export/qif-imp/assistant-qif-import.c

```diff
@@ -1800,9 +1800,30 @@ gnc_ui_qif_import_load_progress_start_cb (GtkButton * button,
 
     /* This step will fill 70% of the bar. */
     gnc_progress_dialog_push (wind->load_progress, 0.7);
-    load_return = scm_call_5 (qif_file_load,
+
+    {
+        gsize length;
+        gchar *contents;
+        gchar *utf8_string;
+        GError *gerror;
+
+        g_file_get_contents(path_to_load, &contents, &length, NULL);
+
+        utf8_string = g_convert (contents, length, "UTF-8",
+                                 qif_encoding, NULL, NULL, &gerror);
+
+        /* bwahaha. crash. */
+        g_assert (gerror == G_CONVERT_ERROR_NO_CONVERSION);
+
```

Comment:
I thought that the whole point of this exercise was to handle `G_CONVERT_ERROR_ILLEGAL_SEQUENCE`, but regardless if you don't handle the other errors you'll crash anyway from passing `NULL` to `scm_from_utf8_string`.

Reply:
Yes the point is to expose (and eventually handle) a conversion error in pure C, and I'll leave the handling & workflow rearrangement to someone more capable than me :)

Reply:
OK, so close this and add a note on the bug that you've realized you're out of your depth on fixing it.